packages/api/schema/stryker-core.json

```diff
@@ -253,6 +262,452 @@
           "default": true
         }
       }
+    },
+    "MutationSpecification": {
+      "anyOf": [
+        { "$ref": "#/definitions/MutationLevelName"},
+        { "$ref": "#/definitions/MutatorDefinition"},
+        { "$ref": "#/definitions/MutatorGroupName"}
+      ]
+    },
+    "MutationLevelName": { 
+      "title": "MutationLevelName", 
+      "type": "string"
+    },
+    "MutatorDefinition" : {
+      "anyOf": [
+        { "$ref": "#/definitions/ArithmeticOperator" },
+        { "$ref": "#/definitions/ArrayDeclaration" },
+        { "$ref": "#/definitions/ArrowFunction"},
+        { "$ref": "#/definitions/AssignmentOperator" },
+        { "$ref": "#/definitions/BlockStatement" },
+        { "$ref": "#/definitions/BooleanLiteral" },
+        { "$ref": "#/definitions/ConditionalExpression" },
+        { "$ref": "#/definitions/EqualityOperator" },
+        { "$ref": "#/definitions/LogicalOperator"},
+        { "$ref": "#/definitions/MethodExpression" },
+        { "$ref": "#/definitions/ObjectLiteral" },
+        { "$ref": "#/definitions/OptionalChaining" },
+        { "$ref": "#/definitions/Regex" },
+        { "$ref": "#/definitions/StringLiteral" },
+        { "$ref": "#/definitions/UnaryOperator" },
+        { "$ref": "#/definitions/UpdateOperator" }
+      ]
+    },
+    "MutatorGroupName": {
+      "enum": [
+        "@ArithmeticOperator",
+        "@ArrayDeclaration",
+        "@AssignmentOperator",
+        "@BooleanLiteral",
+        "@ConditionalExpression",
+        "@EqualityOperator",
+        "@MethodExpression",
+        "@ObjectLiteral",
+        "@OptionalChaining",
+        "@StringLiteral",
+        "@UnaryOperator",
+        "@UpdateOperator"
+      ]
+    },
+    "ArithmeticOperator": {
+      "title": "ArithmeticOperator",
+      "anyOf": [
+        {
+          "const" : "AdditionOperatorNegation",
+          "description": "Replace ```a + b``` with ```a - b```."
```

Comment:
It is very useful to see all the descriptions here. It will help Stryker users enormously. 👏